src/components/organisms/plans/plans.scss

```
@@ -0,0 +1,29 @@
+@use "../../reset" as *;
+@use "../../variables" as *;
+
+.organisms-plans {
+  .message_wrapper {
+    padding-bottom: 32px;
+  }
+  margin-bottom: 100px;
+  @media (max-width: $breakpoint-md) {
+    margin-bottom: 35px;
+  }
+  .plans-wrapper {
+    display: grid;
+    grid-template-columns: auto auto auto;
```

```suggestion
    grid-template-columns: 1fr 1fr 1fr;
```
単位としてfrのほうがよく使うイメージがあるので、こちらも把握しておきましょう！
参考：https://kumonosublog.com/coding/css/grid_layout_fr_auto/